src/DssSpell.t.sol

```diff
@@ -113,23 +88,30 @@ contract DssSpellTest is DSTest, DSMath {
     VatAbstract              vat = VatAbstract(        addr.addr("MCD_VAT"));
     VowAbstract              vow = VowAbstract(        addr.addr("MCD_VOW"));
     CatAbstract              cat = CatAbstract(        addr.addr("MCD_CAT"));
-    DogAbstract              dog = DogAbstract(        addr.addr("MCD_DOG"));
     PotAbstract              pot = PotAbstract(        addr.addr("MCD_POT"));
     JugAbstract              jug = JugAbstract(        addr.addr("MCD_JUG"));
-    SpotAbstract         spotter = SpotAbstract(       addr.addr("MCD_SPOT"));
+    SpotAbstract            spot = SpotAbstract(       addr.addr("MCD_SPOT"));
     DaiAbstract              dai = DaiAbstract(        addr.addr("MCD_DAI"));
-    DaiJoinAbstract      daiJoin = DaiJoinAbstract(    addr.addr("MCD_JOIN_DAI"));
+    DaiJoinAbstract      daiJoin = DaiJoinAbstract(    addr.addr("MCD_DAI_JOIN"));
```

Comment:
I would suggest to leave `MCD_JOIN_DAI` as matches both the `addresses_mainnet.sol` and the Changelog. 
This change will return `address(0)` in the tests.